### ISSUE TITLE
aws command fails in test_external_modification test

### ISSUE THREAD

**Issue by ggtakec** [s3fs-fuse/s3fs-fuse#1105]
### Additional Information
- OS: ubuntu 14.04(trusty)
- on TravisCI

#### Version of s3fs being used (s3fs --version)
3ad1c95e86f0e8ec21a846edf5be675b7d75661a

#### Version of fuse being used (pkg-config --modversion fuse, rpm -qi fuse, dpkg -s fuse)
n/a

#### Kernel information (uname -r)
n/a

#### GNU/Linux Distribution, if applicable (cat /etc/os-release)
n/a

#### s3fs command line used, if applicable
n/a
#### /etc/fstab entry, if applicable
n/a
#### s3fs syslog messages (grep s3fs /var/log/syslog, journalctl | grep s3fs, or s3fs outputs)
n/a
### Details about issue
Fail with TravisCI's ubuntu trusty test_external_modification test.
An error has occurred in the following line:
```
aws s3 --endpoint-url "${S3_URL}" --no-verify-ssl cp - "s3://${TEST_BUCKET_1}/${OBJECT_NAME}"
```

Error details
```
Traceback (most recent call last):
  File "/usr/local/bin/aws", line 19, in <module>
    import awscli.clidriver
  File "/usr/local/lib/python2.7/dist-packages/awscli/clidriver.py", line 36, in <module>
    from awscli.help import ProviderHelpCommand
  File "/usr/local/lib/python2.7/dist-packages/awscli/help.py", line 20, in <module>
    from docutils.core import publish_string
  File "/usr/local/lib/python2.7/dist-packages/docutils/core.py", line 246
    print('\n::: Runtime settings:', file=self._stderr)
                                         ^
SyntaxError: invalid syntax
```

The following error (warning) is also output when installing awscli.
```
The directory '/home/travis/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
 .
 .
 .
DEPRECATION: Uninstalling a distutils installed project (PyYAML) has been deprecated and will be removed in a future version. This is due to the fact that uninstalling a distutils project will only partially uninstall the project.
```

**Comment by ggtakec** [comment]
Additional information

I tried some of the following measures, but for the moment I can not avoid this error.
- try to install and use debian package's awscli
- using "-H" option for sudo
- adding "--ignore-installed six" option for pip install
- clear PYTHONPATH env when run aws command
- set PYTHONPATH env to python2.7

However, this error has not been avoided yet.

**Comment by ggtakec** [comment]
This seems a bug of awscli, it is fixed 1.16.202 awscli.
I will try to test soon.

**Comment by gaul** [comment]
It might make sense to pin a particular version.

**Comment by ggtakec** [comment]
This has been corrected in 1.16.202.
Since awscli support is so fast, version fixing may not be necessary.
As it seems to be all right now, let's leave version unfixed(latest).
